src/Hazelcast.Net.Examples/DistributedObjects/VectorCollectionExample.cs

```diff
@@ -59,7 +77,7 @@ public static async Task Main(string[] args)
 
             // Part 2: Query the collection
             //Arrange the vector values
-            var queryVector = VectorValues.Of(new float[] { 0.1f, 0.2f, 0.3f });
+            var queryVector = VectorValues.Of(new float[] { 0.1f, 0.2f, 0.2f });
```

Comment:
?

Reply:
Just a small change to show capabilities of similarity search. Otherwise, it looks like exact match instead of search.